rest/server_context.go

```diff
@@ -1147,6 +1149,68 @@ func (sc *ServerContext) updateCalculatedStats() {
 
 }
 
+func initClusterAgent(clusterAddress, clusterUser, clusterPass, certPath, keyPath string) (*gocbcore.Agent, error) {
+	authenticator, err := base.GoCBCoreAuthConfig(clusterUser, clusterPass, certPath, keyPath)
+	if err != nil {
+		return nil, err
+	}
+
+	config := gocbcore.AgentConfig{
+		Auth:           authenticator,
+		AuthMechanisms: []gocbcore.AuthMechanism{gocbcore.ScramSha512AuthMechanism},
+		TLSRootCAProvider: func() *x509.CertPool {
+			return nil
+		},
```

Comment:
Doesn't this need to be set in order to trust a cert from both a known and unkown signing authority on the server? I think if you return nil here, _no_ cert will be trusted.

Reply:
I think if I return `nil` it basically skips verify but yeah. Adding this.